supervisor/bootstrap.py

```diff
@@ -222,6 +224,13 @@ def initialize_system(coresys: CoreSys) -> None:
         config.path_addon_configs.mkdir()
 
 
+def warning_handler(message, category, filename, lineno, file=None, line=None):
+    """Warning handler which logs warnings using the logging module."""
+    _LOGGER.warning("%s:%s: %s: %s", filename, lineno, category.__name__, message)
+    if isinstance(message, Exception):
+        capture_exception(message)
```

Comment:
I get what you're trying to do here but I'm not sure its a good idea to merge this before even attempting to tackle the problem of reducing/eliminating dangling tasks that we have right now. This may create a lot of noise in sentry an not end up being super helpful. I think maybe we should attempt to tackle the problem first and then put something like this in to help us catch and clean up any we missed.

Reply:
What I am after here are RuntimeWarning warnings "coroutine was never awaited". From what I can tell, those are not the same as not awaited tasks. I don't think not awaited task cause any runtime warnings. Afaik, they only show up when they raise an exception currently.